app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -64,9 +66,13 @@ protected function validator(array $data)
     protected function create(array $data)
     {
         return User::create([
-            'name' => $data['name'],
-            'email' => $data['email'],
-            'password' => Hash::make($data['password']),
+            'username'      => $data['username'],
+            'name'          => $data['name'],
+            'department'    => $data['department'],
+            'email'         => $data['email'],
+            'role'          => 'none',
```

Comment:
Think we may need a separate roles table and allow a user to have multiple roles. Otherwise we'll end up with checks like this in our templates
```php
 @if (auth::user()->role == "mechanic" || auth::user()->role == "customer_service" || auth::user()->role == "finance" || auth::user()->role == "customer_service_manager")

 ```
 
 Would be nicer if it was something like 
 ```php
  @if (in_array("customer_service", auth::user()->roles) 
 ```
 
 OR leave users with one role and a third table for role permissions
 ```php
   @if (in_array("update-pid", auth::user()->role->permissions) 
 ```

Reply:
The structure was based on Toms data where each user only has one role.